cmd/admin-user-policy.go

```diff
@@ -76,12 +78,34 @@ func mainAdminUserPolicy(ctx *cli.Context) error {
 		fatalIf(probe.NewError(e).Trace(args...), "Unable to fetch user policy document")
 	}
 
-	policies := strings.Split(user.PolicyName, ",")
-	for _, policy := range policies {
-		pinfo, e := getPolicyInfo(client, policy)
-		fatalIf(probe.NewError(e).Trace(args...), "Unable to fetch user policy document for policy "+policy)
-		fmt.Println(string(pinfo.Policy))
+	policyNames := strings.Split(user.PolicyName, ",")
+
+	if len(policyNames) == 1 {
+		policyInfo, err := getPolicyInfo(client, policyNames[0])
+		if err != nil {
+			fatalIf(probe.NewError(err).Trace(), "Unable to fetch user policy document")
+		}
+		fmt.Println(string(policyInfo.Policy))
+		return nil
```

Comment:
```suggestion
	if len(policyNames) == 1 {
		policyInfo, err := getPolicyInfo(client, policyNames[0])
		if err != nil {
			fatalIf(probe.NewError(err).Trace(), "Unable to fetch user policy document")
		}
		fmt.Println(string(policyInfo.Policy))
		return nil
```

This is not needed not sure why we have this.

Reply:
Thanks for the suggestion @harshavardhana `MergePolicies()` works for a policy too